src/Blocks/components/load-more/assets/index.js

```diff
@@ -0,0 +1,33 @@
+import domReady from '@wordpress/dom-ready';
+
+
+
+domReady(() => {
+	let elements = document.querySelectorAll('.load-more');
+
+	let start = 5;
+	let limit = 4;
+
+	for (let element of elements) {
+		const url = element.dataset.url;
+		const containerClass = element.dataset.container;
+		const button = element.querySelector('button');
```

Comment:
all js selectors must have .js- prefix

Reply:
Is `.js-load-more load-more__button` ok for a selector?

Reply:
you should only use .js-load-more

Reply:
In this case I need to target the button so I can add an event listener to it, though.

`.js-load-more` is in the top component div.